src/lib/dom.iterable.d.ts

```diff
@@ -4,6 +4,23 @@ interface DOMTokenList {
     [Symbol.iterator](): IterableIterator<string>;
 }
 
+interface FormData {
+    /**
+     * Returns an array of key, value pairs for every entry in the list
+     */
+    entries(): IterableIterator<[string, any]>;

```

Comment:
from [spec](https://xhr.spec.whatwg.org/#formdataentryvalue), the values are `string | File`; that applies to `value`, `entries` and `[Symbol.iterator]`.